packages/dialtone-vue3/components/popover/popover.vue

```diff
@@ -521,6 +522,29 @@ export default {
       type: Boolean,
       default: false,
     },
+
+    /**
+     * The timer is used only when the hovercard prop is true.
+     * It defines the delays when opening several hovercards.
+     * It must have the keys: enter, leave and current.
+     * If null, the default delay of 300ms will be used.
+     */
+    timer: {
+      type: [Object, null],
+      default: null,
+      validator: timer => {
+        return timer === null || (timer.enter && timer.leave && Object.keys(timer).includes('current'));
+      },
+    },
+
+    /**
+     * A prefix for the data-qa tags in the elements inside the popover.
+     * If null, `dt-popover` will be used.
+     */
+    qa: {
+      type: String,
+      default: null,
+    },
```

Comment:
Can we just pass a standard `data-qa="value"` from the hovercard and then do something like 
```
:data-qa="$attrs['data-qa'] ? `${ $attrs['data-qa'] }-content` : 'dt-popover-content'"
``` 
and so on? The prop feels a little awkward to me but I'm not totally against it if you  want to do it that way.

Reply:
That was fixed in the latest changes :)